### PR TITLE
adds a first set of examples

### PR DESCRIPTION
**Pull request by mariusgiger**

### CODE REVIEW

File: examples/data/data-stream-with-tag.mjs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+const wsBaseUrl = process.env.WS_BASE_URL || "wss://api.akenza.io";
+const apiKey = process.env.API_KEY;
+
+const page = await loadTags(0);
+if (page.totalElements == 0) {
+  console.log("no tags found");
+  process.exit(-1);
+}
+const subscriptions = [{ tagId: page.content[0].id, topic: "*" }];
+const ws = new WebSocket(`${wsBaseUrl}/v3/data-streams?xApiKey=${apiKey}`, {
```
Comment (Biglr) [inline examples/data/data-stream-with-tag.mjs:20]: is this param for api key correct? ^^

File: examples/data/data-stream-with-tag.mjs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+  subscribe();
+});
+
+let messageCounter = 0;
+ws.on("message", function incoming(message) {
+  message = JSON.parse(message);
+  console.log(message);
+  messageCounter = messageCounter + 1;
+  if (messageCounter % 10 === 0) {
```
Comment (Biglr) [inline examples/data/data-stream-with-tag.mjs:55]: why modulo 10? are you only logging every tenth increase?

File: examples/management/get-device.mjs [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  };
+
+  console.log(`getting device for ${url}`);
+  const response = await axios.get(url, options);
+  return response.data;
+}
+
+async function loadDeviceByDeviceId(deviceId) {
+  const url = `${baseUrl}/v3/devices/by-device-id?deviceId=${deviceId}`;
+  const options = {
```
Comment (Biglr) [inline examples/management/get-device.mjs:31]: can these option be extracted somewhere?

File: examples/management/get-devices.mjs [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+import { loadAllEntities, isRunDirectly } from "../utils.mjs";
+config({
+  path: process.env.CONFIG_PATH || path.resolve(process.cwd(), ".env"),
+});
+
+const apiKey = process.env.API_KEY;
+const workspaceId = process.env.WORKSPACE_ID;
+const baseUrl = process.env.BASE_URL || "https://api.akenza.io";
+
+export async function loadDevices(pageNumber, pageSize = 100) {
```
Comment (Biglr) [inline examples/management/get-devices.mjs:16]: maybe also assign a default value to `pageNumber`, also add a comment that paging is 0-based

**Review by Biglr — APPROVED** [review]

File: examples/data/query-data.mjs [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  const options = {
+    headers: {
+      "x-api-key": apiKey,
+      "Content-Type": "application/json",
+    },
+  };
+  const body = {
+    topic: "*",
+    timestamp: {
+      gte: "2022-06-30T00:00:00.000Z",
```
Comment (SavinNiederer) [inline examples/data/query-data.mjs:27]: should't this be variable? its probably ok for now but not in a few month

**Review by SavinNiederer — APPROVED** [review]

**Review by markovip — APPROVED** [review]

File: examples/data/data-stream-with-tag.mjs (hunk 1 above)
Comment (mariusgiger) [reply examples/data/data-stream-with-tag.mjs:20]: yes it is correct, but I agree not the most beautiful thing..

File: examples/data/data-stream-with-tag.mjs (hunk 2 above)
Comment (mariusgiger) [reply examples/data/data-stream-with-tag.mjs:55]: the message counter is only logged every 1th iteration, this is just an example

File: examples/management/get-device.mjs (hunk 3 above)
Comment (mariusgiger) [reply examples/management/get-device.mjs:31]: this is an example and should show how to use the api, therefore being verbose is fine

File: examples/management/get-devices.mjs (hunk 4 above)
Comment (mariusgiger) [reply examples/management/get-devices.mjs:16]: I have added a task to describe pagination in the api docs in the next sprint

File: examples/data/query-data.mjs (hunk 5 above)
Comment (mariusgiger) [reply examples/data/query-data.mjs:27]: it is just an example, I wanted to keep it because it shows the date format

**Pull request merged**